Examples/Tests/PEC/analysis_pec.py

```diff
@@ -0,0 +1,84 @@
+#! /usr/bin/env python
+
+#
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+
+# This is a script that analyses the simulation results from
+# the script `inputs.3d`. This simulates a sinusoindal wave.
```

Comment:
Do you mean `Examples/Tests/PEC/inputs_field_PEC_3d`

Reply:
yes. Thank you. added name of the input file, without the path.